docs/index-hu.md

```diff
@@ -518,18 +518,16 @@ soap.createClient(url, function(err, client) {
 </div>
 <div class="tab-pane fade" id="dotnet" role="tabpanel" aria-labelledby="dotnet-tab" markdown="1">
 
+Az alábbi forráskód bemutatja, hogyan hívhat meg egy webszolgáltatást .NET használatával C# nyelv használatával.
```

Comment:
The correct phrasing in Hungarian: `Az alábbi forráskód bemutatja, hogyan hívhat meg egy webszolgáltatást .NET és C# nyelv használatával.`